### PR TITLE
jarek/7472: ensure deterministic classpath order based on maven 'dependency:build-classpath' goal

### PR DESCRIPTION
**Pull request by jaroslawmalekcodete**

### CODE REVIEW

**Comment by LeeTZ** [comment]
@jpallas Would you mind test it to see if it addresses the issue you mentioned? Thanks

**Comment by jpallas** [comment]
I confirmed that the classpath generated is stable and that it seems to work for several additional Scala libraries that I spot-checked (breeze, vegas, tensorflow_scala).

File: kernel/base/src/test/resources/PomTemplateMagicCommand.xml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jpallas) [inline kernel/base/src/test/resources/PomTemplateMagicCommand.xml]: Unintentional deletion?

Comment (jaroslawmalekcodete) [reply kernel/base/src/test/resources/PomTemplateMagicCommand.xml]: Unintentional deletion reverted

**Pull request merged**